trie/stacktrie.go

```diff
@@ -56,6 +59,8 @@ func NewStackTrie(onTrieNode OnTrieNode) *StackTrie {
 		root:       stPool.Get().(*stNode),
 		h:          newHasher(false),
 		onTrieNode: onTrieNode,
+		kBuf:       make([]byte, 0, 32),
```

Comment:
Afaiu we use kBuf to hold the hex-ed key, and we expect keys to be 32 bytes long on average. Should we maybe start with a 64 byte buffer here?